src/libraries/System.Reflection/tests/AssemblyTests.cs

```diff
@@ -696,6 +696,28 @@ public void GetSatelliteAssemblyNeg()
             Assert.Throws<System.IO.FileNotFoundException>(() => (typeof(AssemblyTests).Assembly.GetSatelliteAssembly(CultureInfo.InvariantCulture)));
         }
 
+        [Fact]
+        public void AssemblyLoadWithPublicKey()
+        {
+            AssemblyName an = new AssemblyName("System.Runtime");
+
+            Assembly a = Assembly.Load(an);
+
+            byte[] publicKey = a.GetName().GetPublicKey();
+            Assert.True(publicKey.Length > 0);
```

Comment:
do the public key and PKT have defined lengths (eg., 8 and 160) and if so, could we assert the lengths of each here?

Reply:
Yes, they do. We have AssemblyName tests that verify that.

This test only requires that the public key and public key token are non-empty to cover the desired scenario.